SoftLayer/CLI/user/notifications.py

```diff
@@ -0,0 +1,34 @@
+"""List user notifications"""
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command()
+@environment.pass_env
+def cli(env):
+    """User Notifications."""
```

Comment:
Change this from `User Notifications` to `My Notifications`.

Reply:
Done.